man/man8/zpool.8

```diff
@@ -435,8 +435,9 @@ See the
 .Sx EXAMPLES
 section for an example of mirroring multiple log devices.
 .Pp
-Log devices can be added, replaced, attached, detached, and imported and
-exported as part of the larger pool.
+Log devices can be added, replaced, attached, detached and removed.  In
+addition, log devices can be imported and exported as part of the pool
```

Comment:
I would suggest `s/can be/are/g`

Reply:
The first sentence describes possible actions, so "can be" is appropriate.  The second sentence describes a mandatory action that's the result of a possible action, so "are" is a better choice.  Thank you.